code/modules/events/infestation.dm

```diff
@@ -131,31 +131,33 @@
 	switch(vermin)
 		if(VERM_MICE)
 			spawn_types = list(/mob/living/simple_animal/mouse/gray, /mob/living/simple_animal/mouse/brown, /mob/living/simple_animal/mouse/white)
-			max_number = 7
+			max_number = 16
+			//Large quantities of mice aren't a potential lag issue since they decompose to
+			//inert skeletons now
 			vermstring = "mice"
 		if(VERM_LIZARDS)
 			spawn_types = list(/mob/living/simple_animal/lizard)
-			max_number = 7
+			max_number = 12
 			vermstring = "lizards"
 		if(VERM_SPIDERS)
 			spawn_types = list(/obj/effect/spider/spiderling)
-			max_number = 3
+			max_number = 5
```

Comment:
There is no major spider event, the spider infestation is a moderate severity one.

Also, spiders spawned from this infestation event have a much slower growth rate, 30% max
In my practical experience with it so far, its never been anything more than a mild inconvenience